examples/apis.json

```diff
@@ -62,7 +62,8 @@
   },
   "motion": {
     "func": "GetPose",
-    "args": []
+    "args": [],
+    "comment":"Note this function requires additional arguments. Please provide valid arguments for function to work properly"
```

Comment:
No context here but why say "this requires additional arguments" and not fill in any default arguments in `"args": []`? Are they like secret arguments or something?

Reply:
thats a good point, let me ask around and see if there are any default variables we can give the user as well